src/components/gameCard/gameCard.tsx

```diff
@@ -0,0 +1,77 @@
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { fas } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+import { ProductInfo } from "@/productInfos";
+import { categoryInfos } from "@/categoriesInfo";
+import getCard from "./getCard";
+
+import styles from "./gameCard.module.scss";
+
+library.add(fas);
+
+interface Mark {
+  value: number;
+}
+
+const StarsMark = (mark: Mark) => (
+  <div className={styles.stars}>{Array(mark.value).fill(<FontAwesomeIcon icon="star" />)}</div>
+);
+
+interface Categories {
+  pc: boolean;
+  xbox: boolean;
+  ps: boolean;
+}
+
+const CategoryMarkers = (categories: Categories) => (
+  <div className={styles.categories}>
+    {categories.pc ? ( // PC category marker
+      <img src={categoryInfos.find((info) => info.name === "PC")?.image} alt="PC.jpg" className={styles.category} />
+    ) : (
+      <span />
```

Comment:
what is it?

Reply:
this copy-paste code is now replaced with map function